typings/copy.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-copy-to-clipboard';
```

Comment:
I wasn't sure where this declaration file belonged, so for now it's living in the same directory as the highlight module declarations.

Reply:
This is great 👍